src/svgdevicecontext.cpp

```diff
@@ -974,6 +976,23 @@ void SvgDeviceContext::AppendIdAndClass(std::string gId, std::string baseClass,
     m_currentNode.append_attribute("class") = baseClass.c_str();
 }
 
+void SvgDeviceContext::AppendAdditionalAttributes(Object *object)
+{
+    std::string baseClass = object->GetClassName();
+    std::transform(baseClass.begin(), baseClass.begin() + 1, baseClass.begin(), ::tolower);
```

Comment:
As far as I can see you don't use this anymore. Please remove

Reply:
True; have overlooked that. Removed now.